src/Commands/make/CmdCommand.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace Blacksmith\Commands\Make;
+
+use Blacksmith\Command;
+
+/**
+ * The "Make Command" Command.
+ *
+ * This command is responsible for creating new commands that will be placed in
+ * the commands directory in the target project's main directory. If provided 
+ * subdirectories to create the command in, the CmdCommand class will place the
+ * new command in those subdirectories.
+ */
+
+class CmdCommand extends Command {
+
+    // Set the signature for the command
+    const signature       = 'make:cmd';
+
+    // Set the description of the command
+    const description     = 'Creates a new command';
+
+    /**
+     * The full path for the new command.
+     * @var string
+     */
+    protected $command_dir_path = '';
+
+    /**
+     * Main method that will run the
+     * command operation.
+     */
+    public function run()
+    {
+        // Validate argument was passed
+        if (empty($this->getArguments())) {
+            $this->console->output->alert('make:cmd requires an argument.');
+            return;
+        }
+
+        $this->createCommand();
+    }
+
+    /**
+     * Create a new command.
+     *
+     * Gets the provided arguments and creates subdirectories if they do not already exist
+     * for the provided command. The last element in the chain of subdirectories is treated
+     * as the name of the callabale command, in which we create a template command file for.
+     *
+     * For example:
+     *
+     *      php blacksmith make:cmd this:is:my:command
```

Comment:
this example is not accurate because you're exploding by DIRECTORY_SEPARATOR and not by :